site/vercel.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can we also add the rewrite rules for routes ending .html here as well?

Reply:
Tried a regex in rewrites and it didn't work. Added the `cleanUrls` setting and is working great now in preview.

Reply:
oh nice!